src/lib/ngsiNotify/QueueNotifier.cpp

```diff
@@ -100,8 +101,8 @@ void QueueNotifier::sendNotifyContextRequest(NotifyContextRequest* ncr, const st
     spathList = "";
   }
 
-  ci.outFormat = format;
-  std::string payload = ncr->render(&ci, NotifyContext, "");
+  ci.outFormat = JSON;
```

Comment:
Maybe the abuse of "format" is making things complicated...

Thus, at the end probably we need two enumaration and my original idea of joining them is not good. One named NotificationFormat (the one about which this PR is mainly about). The other maybe should be MimeType (or something like that) as their values are JSON and Text vay the time being. I think is a bad idea mixing into the same type, as that would mix different kind of objects.